migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,158 @@
+
+Review the changes and other considerations that might affect your migration from {product-title} 3.11 to {product-title} 4.
+
+[id="migration-preparing-storage"]
+=== Storage considerations
+
+Review the storage changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== TODO
+
+[id="migration-preparing-networking"]
+=== Networking considerations
+
+Review the networking changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Network isolation mode
+
+The default network isolation mode for {product-title} 3.11 was `ovs-subnet`. The default network isolation mode for {product-title} 4.2 is now NetworkPolicy. If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant` mode, it is recommended to switch to the NetworkPolicy mode for your {product-title} 4.2 cluster. If you want to maintain the `ovs-multitenant` behavior while using NetworkPolicy in {product-title} 4.2, follow the steps to xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-multitenant-isolation_configuring-networkpolicy-plugin[configure multitenant isolation using NetworkPolicy].
+
+For more information, see xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-about_configuring-networkpolicy-plugin[About network policy].
+
+[discrete]
+==== Encrypting traffic between hosts
+
+In {product-title} 3.11, you could use IPsec to encrypt traffic between hosts. {product-title} 4.2 does not support IPsec. It is recommended to use Red Hat OpenShift Service Mesh to enable mutual TLS between services.
+
+For more information, see xref:../service_mesh/service_mesh_arch/understanding-ossm.adoc#understanding-ossm[Understanding Red Hat OpenShift Service Mesh].
+
+[id="migration-preparing-logging"]
+=== Logging considerations
+
+Review the logging changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Deploying cluster logging
+
+{product-title} 4 provides a simple deployment mechanism for cluster logging, by using a Cluster Logging custom resource. Once deployed, the cluster logging experience is the same as it was in {product-title} 3.11.
+
+For more information, see xref:../logging/cluster-logging-deploying-about.adoc#cluster-logging-deploying-about_cluster-logging-deploying-about[About deploying and configuring cluster logging].
+
+[discrete]
+==== Aggregated logging data
+
+You cannot transition your aggregate logging data from {product-title} 3.11 into your new {product-title} 4 cluster. You must start fresh with the new cluster.
+
+For more information, see xref:../logging/cluster-logging.adoc#cluster-logging-about_cluster-logging[About cluster logging].
+
+[id="migration-preparing-security"]
+=== Security considerations
+
+Review the security changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Unauthenticated access to discovery endpoints
```

Comment:
@sttts Can you please review this section to make sure it's accurate? And that we're not missing anything else? (@stlaz is going to help me add some oauth info as well). Thanks!

Preview: https://migration-guide--ocpdocs.netlify.com/openshift-enterprise/latest/migration/planning-migration-3-to-4.html#migration-preparing-security